docs/guides/organization.md

```diff
@@ -1,17 +1,37 @@
 # Organization
-[Organization Interface](../api/interfaces/cacheServerClient_cacheServerClient_types.IOrganization.md)  
 
-/*
-TO DO: 
-//Define Organization in context of iam-client-lib/Switchboard
-//Define Suborganization in context of iam-client-lib/Switchboard
+[Organization Interface](../api/interfaces/cacheServerClient_cacheServerClient_types.IOrganization.md)
 
-//Give an example(s) of an organization
-//Give an example(s) of a sub-organization
-*/
+**Organization** is basic type in EnergyWeb domains tree. Its main purpose is to be a host.
```

Comment:
What do you mean by "be a host"?

Reply:
I tried to express that organization serves as container or root which other domains attached to. May be there is more appropriate term for this?

Reply:
I think that either of the terms, "container" or "root", would be good, however how about the text below?
```suggestion
**Organization** is basic type in EnergyWeb domains tree. Its main purpose is to be a root that can group other domains, such as apps and roles.
```